app/components/number-widget/component.js

```diff
@@ -7,6 +7,7 @@ export default Ember.Component.extend({
     settings : {
         fontSize: 3,
         fontColor: '#F44336',
+        noLink: false,
```

Comment:
Yes, but I would then have to add link true to all search widgets. And this way it's an opt out so I thought it would fit most cases rather than have an opt in and require all number widgets to take care of it. What do you think?